examples/plugins-hookable/README.md

```diff
@@ -0,0 +1,3 @@
+# Using parallel and hookable plugins with Nuxt.js
```

Comment:
Can we simplify new concept to just hooks? (instead of hookable)

Reply:
We could simplify hable to also export individual functions, then have the Hookable class use those functions too? Then we could do here:
```
import { hook, callHook } as Hookable from 'hable'
```
and the unused functionality of hable would be tree-shaked

Reply:
indeed. functions are always more js-native than classes IMO :D Anyway this comment was just regarding naming...